cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java

```diff
@@ -200,15 +200,17 @@ private void logKeyStore(KeyStore ks, String ksLocation, char[] ksPwd) {
       try {
         for (Enumeration<String> e = ks.aliases(); e.hasMoreElements(); ) {
           String alias = e.nextElement();
-          Key key = ks.getKey(alias, ksPwd);
```

Comment:
this change will break keystores with passwords.

Reply:
please add unit test with keystore that contains password protected cert that expires in 100~ years

Reply:
> this change will break keystores with passwords.

Are you sure of this ? I believe we tried with password protect keystores. Do you have any example of unit tests ?

Reply:
I'll grab an example on Monday when I'm connected to work VPN, recently integrated password protected certificates in one of our apps.

Reply:
@johnou thanks a lot!

Reply:
@johnou ping

Reply:
@pierventre apologies for the satellite delay.. https://gist.github.com/johnou/d9118891ada6791717ea322242cd497a